### ISSUE TITLE
How do netlify works

### ISSUE THREAD

**Issue by Ezimoha** [iralecode/umarsite#1]
- I don't understand

**Comment by iralecode** [comment]
yes  you have a github whenever you upload a respository it will reflect on netlify you have to have a netlify account too

**Comment by Ezimoha** [comment]
How do I sign up

**Comment by iralecode** [comment]
goto netlify official website and you will see sign in with github that all
anyother thing message me again
